<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addRemoveRooms</class>
 <widget class="QDialog" name="addRemoveRooms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add / Remove Rooms</string>
  </property>
  <widget class="QLabel" name="bg_white">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(154, 140, 152);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bg_dark">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(74, 78, 105);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);
border-radius: 0;</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);
border-radius: 0;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QComboBox" name="idComboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Customer ID:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="addRadio">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Room</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="removeRadio">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Remove Room</string>
   </property>
  </widget>
  <widget class="QLabel" name="customerNameLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="text">
    <string>Customer Full Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="roomComboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="chooseLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose Room:</string>
   </property>
  </widget>
  <zorder>bg_dark</zorder>
  <zorder>bg_white</zorder>
  <zorder>updateButton</zorder>
  <zorder>backButton</zorder>
  <zorder>idComboBox</zorder>
  <zorder>idLabel</zorder>
  <zorder>addRadio</zorder>
  <zorder>removeRadio</zorder>
  <zorder>customerNameLabel</zorder>
  <zorder>roomComboBox</zorder>
  <zorder>chooseLabel</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>idComboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>customerNameLabel</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
